src/audio/audiooutput.cc

```diff
@@ -23,18 +19,7 @@ static QAudioFormat format( int sampleRate, int channelCount )
 
   out.setSampleRate( sampleRate );
   out.setChannelCount( channelCount );
-#if QT_VERSION < QT_VERSION_CHECK( 6, 0, 0 )
-  out.setByteOrder( QAudioFormat::LittleEndian );
-  out.setCodec( QLatin1String( "audio/pcm" ) );
```

Comment:
PCM, commenly used in `.wav` -> https://en.wikipedia.org/wiki/Pulse-code_modulation